File: R/app/src/main/java/com/example/voiceassistent/messageview/Message.java
package com.example.voiceassistent.messageview;

import java.util.Date;

public class Message {
    public String text;
    public Date date;
    public Boolean isSend;

    public Message(String text, Boolean isSend){
        this.text = text;
        this.date = new Date();
        this.isSend = isSend;
    }
    public Message(MessageEntity entity) {
        this.text = entity.text;
        if (entity.date == null)
            this.date = null;
        else
            this.date = new Date(entity.date);
        this.isSend = entity.isSend != 0;
    }

}

File: Rkt/app/src/main/java/com/example/voiceassistent/messageview/Message.kt
package com.example.voiceassistent.messageview

import java.util.*

class Message {
    @JvmField
    var text: String
    @JvmField
    var date: Date? = null
    @JvmField
    var isSend: Boolean?

    constructor(text: String, isSend: Boolean) {
        this.text = text
        date = Date()
        this.isSend = isSend
    }

    constructor(entity: MessageEntity) {
        text = entity.text
        if (entity.date == null) date = null else date = Date(entity.date)
        isSend = entity.isSend != 0
    }
}